### PR TITLE
Doxygen documentation for STEER/AOD/*

### PR DESCRIPTION
**Pull request by mikesas**

### CODE REVIEW

File: STEER/AOD/AliAODJet.h [hunk 1]
```diff
@@ -117,16 +117,20 @@ class AliAODJet : public AliVParticle {
...
-    Double32_t      fPtSubtracted[2];        //[0,0,12] pT after subtraction can be negative four momentum close to 0 in this case, 12 bit precision
-    Double32_t      fPtLeadingConstituent;   //[0,0,12] pT of leading constituent
-    UInt_t          fTrigger;                // Bit mask to flag jets triggered by a certain detector  
-    TLorentzVector* fMomentum;               // Jet 4-momentum vector
-    TLorentzVector* fVectorAreaCharged;      // jet area four momentum 
-    TRefArray*      fRefTracks;              // array of references to the tracks belonging to the jet
+    Double32_t      fBackgEnergy[2];         ///< Subtracted background energy
+    Double32_t      fEffectiveArea[2];       ///< Effective jet area used for background subtraction
+    /// relative error of jet areas, 10 bit precision
+    Double32_t      fEffectiveAreaError[2];  //[0,1,10]
```
Comment (dberzano) [inline STEER/AOD/AliAODJet.h]: And this is fine too, right? I am wondering if there's a way to have this Doxygen-documented, and at the same time. I haven't, unfortunately, found anything even inside ROOT 6. They seem to just drop the documentation. I think that breaking it in two lines is probably the best, just checking...

File: STEER/AOD/AliAODTagCreator.cxx [hunk 2]
```diff
@@ -63,14 +57,16 @@ ClassImp(AliAODTagCreator)
 
 //______________________________________________________________________________
 AliAODTagCreator::~AliAODTagCreator() {
-//================Default destructor for a AliAODTagCreator===================
+/// ================Default destructor for a AliAODTagCreator===================
```
Comment (dberzano) [inline STEER/AOD/AliAODTagCreator.cxx]: I'd remove the `=` from this comment.

**Comment by dberzano** [comment]
I really went through ALL the data members and they all seem to be fine to me. It took a while though :-)

File: STEER/AOD/AliAODHeader.h [hunk 3]
```diff
@@ -228,64 +228,65 @@ class AliAODHeader : public AliVAODHeader {
...
+  Double32_t  fEventplaneQx;        ///< Q vector component x from TPC event plance
+  Double32_t  fEventplaneQy;        ///< Q vector component y from TPC event plance
+  Double32_t  fZDCN1Energy;         ///< reconstructed energy in the neutron1 ZDC
+  Double32_t  fZDCP1Energy;         ///< reconstructed energy in the proton1 ZDC
+  Double32_t  fZDCN2Energy;         ///< reconstructed energy in the neutron2 ZDC
+  Double32_t  fZDCP2Energy;         ///< reconstructed energy in the proton2 ZDC
+  Double32_t  fZDCEMEnergy[2];      ///< reconstructed energy in the electromagnetic ZDCs
+  Int_t       fNQTheta;             ///< number of QTheta elements
+  ///< values to store Lee-Yang-Zeros
+  Double32_t *fQTheta;              //[fNQTheta]
```
[the remark this answers is not in the document]
Comment (mikesas) [reply STEER/AOD/AliAODHeader.h]: Hi Dario,
Yeah we agreed to put the comment one line before and keep the streamer as done above.
Cheers, Mike

File: STEER/AOD/AliAODJet.h (hunk 1 above)
Comment (mikesas) [reply STEER/AOD/AliAODJet.h]: Hi Dario,
Yeah I also think that breaking it up in two lines is the best.
Cheers, Mike

**Comment by dberzano** [comment]
@mikesas could you please rebase it instead of merging? Just run:

```
git pull --rebase origin master
```

to get rid of the `Merge branch master into master` commit. Then force-push.

**Comment by dberzano** [comment]
+1 @mikesas @marcovanleeuwen thank you very much for your patience and your work!

**Pull request merged**